documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc

```diff
@@ -0,0 +1,28 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-exporter.adoc
+[id='con-metrics-kafka-exporter-alerts{context}']
+
+= Kafka Exporter alerting rule examples
+
+If you performed the steps to introduce metrics to your deployment, you will already have your Kafka cluster configured to use the alert notification rules that support Kafka Exporter.
+
+The rules for Kafka Exporter are defined in `prometheus-rules.yaml`, and are deployed with Prometheus.
+For more information, see  xref:assembly-metrics-prometheus-{context}[Prometheus].
+
+The alert notification rules specific to Kafka Exporter are as follows:
+
+`UnderReplicatedPartition`:: An alert to warn that a topic is under-replicated and the broker is not replicating enough partitions.
+A planned restart of the Kafka broker may be required to restart the replication process.
+
+`TooLargeConsumerGroupLag`:: An alert to warn that the lag on a consumer group is too big for a specific topic partition.
```

Comment:
I would mention the default value here too. If I am not wrong, it should be '1000'.